.github/workflows/generate-playground.yml

```diff
@@ -1,17 +1,45 @@
-name: Deploy Chinese
+name: Generate Playground Files
 
 # Controls when the action will run. Triggers the workflow on push or pull request
 # events but only for the master branch
 on:
   push:
     branches: [master]
-  pull_request:
-    branches: [master]
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   # This workflow contains a single job called "build"
-  build:
+  generate-english-playground:
+    # The type of runner that the job will run on
+    runs-on: macos-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Checkout to chinese branch & Generate Chinese
+      - name: Generate English Readme
+        run: |
+          ./generate-playground.sh
```

Comment:
You can do `./generate-playground-cn.sh` here, then the `README-CN.md` can also commit to master.